tests/test_run_haplotagphase.py

```diff
@@ -0,0 +1,45 @@
+"""
+Tests for whatshap extend module
+
+"""
+
+from whatshap.cli.haplotagphase import compute_votes, run_haplotagphase
+from whatshap.core import Read
+
+
+def test_extend():
+    run_haplotagphase(
+        variant_file="tests/data/pacbio/variants.vcf",
+        alignment_file="tests/data/pacbio/haplotagged.bam",
+        reference="tests/data/pacbio/reference.fasta",
+        output="/dev/null",
+    )
```

Comment:
I checked that I phased most of the variants. This occurs because, for some variants, we observe a tie that we do not wish to break with the default parameters. For example, if we set the threshold for votes to 0, we assign only 1 variant vs 3 in `tests/data/pacbio/phased.vcf`.